ideasbox/views.py

```diff
@@ -15,9 +16,11 @@
 def index(request):
     contents = Content.objects.published()[:3]
     random_book = Book.objects.available().order_by('?').first()
+    random_doc = Document.objects.order_by('?').first()
```

Comment:
I don't know for sqlite but generally it's not recommended to sort sql data by "random"